client/src/components/ClientProfile/Information/Information.js

```diff
@@ -16,96 +13,190 @@ import VisibilityIcon from '@material-ui/icons/Visibility';
 import VisibilityOffIcon from '@material-ui/icons/VisibilityOff';
 import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import { KeyboardDatePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
-// Local changes
+// Modal sweetalert2
+import Swal from 'sweetalert2';
+import withReactContent from 'sweetalert2-react-content';
+// Local imports
 import { useStyles } from './style';
-import { dispatchGetCustomer } from '../../../store/auth/actions';
+import './styleModal.css';
+import Tungsten from '../../../fonts/Tungsten-Book.woff';
+import { UPDATE_PASSWORD, UPDATE_CUSTOMER } from '../../../axios/endpoints';
 
+const tungsten = {
+  fontFamily: 'Tungsten Book',
+  fontStyle: 'normal',
+  src: `
+    local('Tungsten Book'),
+    url(${Tungsten}) format('woff')
+  `,
+};
 
-export const Info = (props) => {
+export const Information = (props) => {
   const classes = useStyles();
-  const [expanded, setExpanded] = useState('panel1');
-  const [customerInfo, setCustomerInfo] = useState('customer');
-  const [passwordHidden, setPasswordVisible] = useState(true);
-  const PasswordIcon = passwordHidden ? VisibilityIcon : VisibilityOffIcon;
-
   const inputLabel = React.useRef(null);
-
-  const [values, setValues] = useState({
+  const customerEmpty = {
     firstName: '',
     lastName: '',
-    phone: '',
+    telephone: '',
```

Comment:
What's difference?)

Reply:
The purpose is to have information in line with DB.